runtime/src/accounts.rs

```diff
@@ -749,16 +755,17 @@ impl Accounts {
     }
 
     pub fn new(fork: Fork, in_paths: Option<String>) -> Self {
-        let paths = if in_paths.is_none() {
-            Self::make_default_paths()
+        let (paths, own_paths) = if in_paths.is_none() {
+            (Self::make_default_paths(), true)
```

Comment:
which code doesn't pass `in_paths, now?

Reply:
Basically all the tests which create banks, in_paths is really only populated when the --accounts argument is passed to fullnode from the command line.